pyart/graph/radardisplay.py

```diff
@@ -1007,11 +1035,14 @@ def _label_axes_ray(self, axis_labels, field, ax):
         else:
             ax.set_ylabel(y_label)
 
-    def _label_axes_vpt(self, axis_labels, ax):
+    def _label_axes_vpt(self, axis_labels, time_axis_flag, ax):
```

Comment:
The 'test_radardisplay_user_specified_labels' test in test_radar_display.py assumes this method takes 3 parameters.  It should be updates with the new call signature (add False before the ax parameter).